File: src/028/028.swift
func solve() -> Int {
    return stride(from: 3, to: 1003, by: 2).reduce(1) { (acc: Int, x: Int) in
        acc + (4 * x * x - (6 * (x - 1)))
    }
}

let result = solve()
print(result)

File: src/028/028.py
#!/usr/bin/env python3
def solve():
    acc = 1
    for x in range(3, 1003, 2):
        acc += 4 * x * x - (6 * (x - 1))
    return acc


if __name__ == '__main__':
    result = solve()
    print(result)
